src/open_inwoner/openzaak/tests/test_notification_utils.py

```diff
@@ -65,7 +60,7 @@ def test_send_case_update_email(self):
         body_html = email.alternatives[0][0]
         self.assertIn(case.identificatie, body_html)
         self.assertIn(case.zaaktype.omschrijving, body_html)
-        self.assertIn("My Translated Status", body_html)
+        self.assertIn(status.statustype.omschrijving, body_html)
```

Comment:
Are we showing the `omschrijving` or the `statustekst`? They could be the same for some statustypes, different for others.

Reply:
Yes you are right, statustekst has priority, I updated the test